tests/e2e-scality/e2e-tests/README.md

```diff
@@ -0,0 +1,116 @@
+# End-to-End Tests for Scality S3 CSI Driver
```

Comment:
We will eventually use K8s e2e framework in the next PR and replace the tests. Currently they are some sample tests to verify the workflow